src/components/metadataEditor/metadataEditor.js

```diff
@@ -21,7 +21,7 @@ import ServerConnections from '../ServerConnections';
 import toast from '../toast/toast';
 import { appRouter } from '../router/appRouter';
 import template from './metadataEditor.template.html';
-import { SeriesStatus } from '@jellyfin/sdk/lib/generated-client';
+import { BaseItemKind, SeriesStatus } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
To ensure tree-shaking can happen during the build, we should use the full path for these imports `@jellyfin/sdk/lib/generated-client/models/base-item-kind` (I think).